### DISCUSSION TITLE
How do I ignore a static file at compile time

### DISCUSSION THREAD

**Discussion by TryChai** [webpack/webpack#16778] [Q&A]
## Feature request

**What is the expected behavior?**
 ignore a static file at compile time

**What is motivation or use case for adding/changing the behavior?**
when we development  is hard to change static file like image ,font ,is not be compile very time 

**How should this be implemented in your opinion?**
add a option to ignore the static file 

**Are you willing to work on this yourself?**
yes

**Comment by alexander-akait** [comment]
What do you mean ignore? Ignore to watch files? Please clarify
